### ISSUE TITLE
Clear chorionic fluid

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7082]
**Preferred term label:**
Clear chorionic fluid

**Synonyms**

**Definition (free text, please give PubMed ID)**
Chorionic cavity with clear fluid

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ichthyosis prematurity syndrome | 608649
PMID:21856041
no frequency

**Your nano-attribution (ORCID)**

**Comment by patt1morgan** [comment]
Hi Peter, 
Echo-free or clear chorionic fluid is a normal ultrasound finding before about 16 weeks gestational age when the amnion/ chorion are still seen as separate cavities.  Normally, after the membranes fuse at @14-16 weeks, the chorionic cavity is not seen unless there is an abnormality causing separation of the membranes. In the case of ichthyosis, the abnormality is that debris is seen in the amniotic cavity, but the fluid in the chorionic cavity, now visualized because of the separation of the membranes, appears echo-free or clear.  Debris, or particles are seen within the 
amniotic fluid and may represent a normal variant, particulate matter, vernix caseosa, intraamniotic blood, or intrauterine meconium passage. (a new term may be required for debris in the amniotic fluid/cavity??)
I hope this is helpful!

**Comment by pnrobinson** [comment]
This is probably not a valid term then, closing, thanks!
